kde-misc/plasma-applet-active-window-control/plasma-applet-active-window-control-1.7.3.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils kde5-functions
+
+DESCRIPTION="Plasma 5 applet for controlling currently active window"
+HOMEPAGE="https://store.kde.org/p/998910/ \
+	https://github.com/kotelnik/plasma-applet-active-window-control"
+
+if [[ ${PV} == 9999 ]] ; then
```

Comment:
You can drop the ${PV} == 9999 block entirely, kde5.eclass will do that for you.